features/login/impl/src/main/res/values-fr/translations.xml

```diff
@@ -22,6 +23,7 @@
   <string name="screen_login_error_deactivated_account">"Ce compte a été désactivé."</string>
   <string name="screen_login_error_invalid_credentials">"Nom d’utilisateur et/ou mot de passe incorrects"</string>
   <string name="screen_login_error_invalid_user_id">"Il ne s’agit pas d’un identifiant utilisateur valide. Format attendu : « @user:homeserver.org »"</string>
+  <string name="screen_login_error_refresh_tokens">"Ce serveur est configuré pour utiliser des tokens d\'actualisation. Ils ne sont pas pris en charge lors de l\'utilisation d\'une connexion basée sur un mot de passe."</string>
```

Comment:
@gaelledel I usually prefer to use `’` instead of `'` for the  apostrophe, it's more correct for French. Can you update the value on Localazy please?